mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_gtid.test

```diff
@@ -0,0 +1,49 @@
+--echo
+--echo # MDEV-34614 mysqlbinlog warn on EOF before GTID in --stop-position
+--echo
+
+--source include/have_binlog_format_row_or_statement.inc
```

Comment:
I originally had
```suggestion
--source include/have_log_bin.inc
```
But (at least for me,) the Mixed format would rotate `0-1-2` and latter to `master-bin.000002`.
`mariadb-binlog path/to/master-bin.000002` somehow even alerted
> ERROR: Found out of order GTID. Got 0-1-1 after 0-1-3

Reply:
Did your test look different when you saw those errors? I can't reproduce them.

_BTW I didn't know about the suggestion code block feature, that is nice :)_

Reply:
I can’t reproduce by running the test by itself either, but it occurs when running the complete `binlog` suite (`mtr --suite=binlog`).

Reply:
Ah, the test just isn't initializing its state properly. You should be able to do
```suggestion
--source include/have_log_bin.inc
RESET MASTER;
```

Note we don't `RESET MASTER` at the end of a test, see the following comment in `rpl_end.inc` for details:
```
# It does not execute RESET MASTER or RESET SLAVE, because that would
# remove binlogs which are possibly useful debug information in case
# the test case later fails with a result mismatch.
```